pkg/controller/perconaservermongodb/backup.go

```diff
@@ -154,7 +154,7 @@ func (r *ReconcilePerconaServerMongoDB) deleteOldBackupTasks(ctx context.Context
 
 			}
 		} else {
-			log.Info("deleting outdated backup job", "cluster", cr.Name, "name", item.Name, "namespace", cr.Namespace)
+			log.Info("deleting outdated backup job", "name", item.Name)
```

Comment:
don't we already have `name` key for `cr.Name`? it'll be duplicated, no?

Reply:
Good catch @egegunes , yes, this would override cluster name basically :)

Reply:
Fixed: [9654ecd](https://github.com/percona/percona-server-mongodb-operator/pull/1141/commits/9654ecda3ede25ef40f23fc63e83a1a0cec363fe)